pkgs/by-name/pd/pdftowrite/package.nix

```
@@ -0,0 +1,80 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  makeWrapper,
+  versionCheckHook,
+  nix-update-script,
+
+  # shared
+  gzip,
+  # pdftowrite
+  poppler-utils,
+  inkscape,
+  ghostscript,
+  imagemagick,
+  libxml2,
+  libxslt,
+  # writetopdf
+  wkhtmltopdf,
+  pdftk,
+  librsvg,
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "pdftowrite";
+  version = "2021.05.03";
+
+  src = fetchFromGitHub {
+    owner = "apebl";
+    repo = "pdftowrite";
+    tag = version;
+    hash = "sha256-IFX9K74tfGKyMtqlc/RsV00baZEzE3HcPAGfrmTHnDQ=";
+  };
+
+  dependencies = [
+    # shared
+    gzip
+    python3Packages.shortuuid
+    python3Packages.picosvg
+    # pdftowrite
+    poppler-utils
+    inkscape
+    ghostscript
+    imagemagick
+    libxml2
+    libxslt
+    # writetopdf
+    wkhtmltopdf
+    pdftk
+    librsvg
+
+    makeWrapper
+  ];
+
+  build-system = [
+    python3Packages.setuptools
+    python3Packages.setuptools-scm
+  ];
```

```suggestion
  dependencies = [
    python3Packages.shortuuid
    python3Packages.picosvg
  ];

  build-system = [
    python3Packages.setuptools
    python3Packages.setuptools-scm
    makeWrapper
  ];
```
dependencies is only for Python packages. See next comment for giving the program access to the programs.